Sources/Operators/ToAsync.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  ToAsync.swift
+//  CombineExt
+//
+//  Created by Thibault Wittemberg on 2021-06-15.
+//  Copyright © 2021 Combine Community. All rights reserved.
+//
+
+import Combine
+
+#if canImport(Combine)
+import Combine
+
+@available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *)
+public extension Future {
+    func toAsync() async -> Output where Failure == Never {
+        var subscriptions = [AnyCancellable]()
+
+        return await withUnsafeContinuation { [weak self] (continuation: UnsafeContinuation<Output, Never>) in
```

Comment:
Why unsafe continuation? Seems like a classic checked throwing case here

Reply:
As discussed together, although withUnsafeContinuation seems "safe" to use with a Future, we'll go on with a "withCheckedContinuation" as we don't want to provide an API that could lead to an inconsistant state. That way, if there are multiple calls to "resume", there will be a runtime error.